hosting/templates/hosting/order_confirm.html

```diff
@@ -0,0 +1,144 @@
+{% extends "hosting/base_short.html" %}
+{% load staticfiles i18n custom_tags %}
+
+{% block content %}
+<div id="order-detail{{order.pk}}" class="order-detail-container">
```

Comment:
Not sure if `{{order.pk}}` serves any purpose in this case and same with setting the model in views.py as we have not created any HostingOrder yet